openedx/adg/lms/webinars/tests/test_admin.py

```diff
@@ -170,34 +172,72 @@ def test_send_update_emails_to_registrants_in_webinar_update(webinar_admin_insta
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('update', (True, False))
-def test_save_related_send_emails(request, webinar_admin_instance, webinar, mocker, update):
+@pytest.mark.parametrize('is_update, is_already_published, is_set_to_be_published', [
+    (False, False, False),
+    (False, False, True),
+    (True, False, True),
+    (True, True, True)
+])
+def test_save_related_send_emails(
+    request, webinar_admin_instance, mocker, is_update, is_already_published, is_set_to_be_published
+):
     """
-    Test that upon creating/updating webinar, emails for invites/updated invites are sent.
+    Test that the webinar updation or creation emails are only sent when a webinar is published.
```

Comment:
In the docstring it states "webinar **updation** or creation emails..." while in the test function we are only testing for creation emails. I think the code to test updation emails was removed in this second iteration and the docstring was left unchanged. If you think that the check for updation is not necessary, then please update the docstring as well.

Reply:
Basically, I meant when a webinar is updated or created not that the updation and creation emails are being tested. I'll update it to remove this ambiguity.